From 5854b7bb172f407e2cbb934ec6742828a4bb2d40 Mon Sep 17 00:00:00 2001
From: Radostin Stoyanov <rstoyanov1@gmail.com>
Date: Wed, 11 Oct 2017 12:35:53 +0100
Subject: [PATCH 07/45] Replace "Queue" with "queue"

The Queue module has been renamed to queue in Python 3. [1]

[1] https://docs.python.org/2/library/queue.html
---
 virtManager/engine.py     | 12 ++++++------
 virtManager/inspection.py |  2 +-
 virtManager/sshtunnels.py |  4 ++--
 3 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/virtManager/engine.py b/virtManager/engine.py
index c9b1851f..b0b90065 100644
--- a/virtManager/engine.py
+++ b/virtManager/engine.py
@@ -20,7 +20,7 @@
 
 import logging
 import re
-import Queue
+import queue
 import threading
 import traceback
 
@@ -105,7 +105,7 @@ class vmmEngine(vmmGObject):
                                             target=self._handle_tick_queue,
                                             args=())
         self._tick_thread.daemon = True
-        self._tick_queue = Queue.PriorityQueue(100)
+        self._tick_queue = queue.PriorityQueue(100)
 
         self.inspection = None
         self._create_inspection_thread()
@@ -272,15 +272,15 @@ class vmmEngine(vmmGObject):
         """
         We serialize conn autostart, so polkit/ssh-askpass doesn't spam
         """
-        queue = Queue.Queue()
+        connections_queue = queue.Queue()
         auto_conns = [uri for uri in self.conns
                       if self.conns[uri]["conn"].get_autoconnect()]
 
         def add_next_to_queue():
             if not auto_conns:
-                queue.put(None)
+                connections_queue.put(None)
             else:
-                queue.put(auto_conns.pop(0))
+                connections_queue.put(auto_conns.pop(0))
 
         def state_change_cb(conn):
             if conn.is_active():
@@ -292,7 +292,7 @@ class vmmEngine(vmmGObject):
 
         def handle_queue():
             while True:
-                uri = queue.get()
+                uri = connections_queue.get()
                 if uri is None:
                     return
                 if uri not in self.conns:
diff --git a/virtManager/inspection.py b/virtManager/inspection.py
index f522e250..810803c7 100644
--- a/virtManager/inspection.py
+++ b/virtManager/inspection.py
@@ -17,7 +17,7 @@
 # MA 02110-1301 USA.
 #
 
-from Queue import Queue
+from queue import Queue
 from threading import Thread
 import logging
 
diff --git a/virtManager/sshtunnels.py b/virtManager/sshtunnels.py
index 0a3b729c..c89bfcd5 100644
--- a/virtManager/sshtunnels.py
+++ b/virtManager/sshtunnels.py
@@ -20,7 +20,7 @@
 import functools
 import logging
 import os
-import Queue
+import queue
 import socket
 import signal
 import sys
@@ -129,7 +129,7 @@ class _TunnelScheduler(object):
     """
     def __init__(self):
         self._thread = None
-        self._queue = Queue.Queue()
+        self._queue = queue.Queue()
         self._lock = threading.Lock()
 
     def _handle_queue(self):
-- 
2.16.2

